app/controllers/application.js

```diff
@@ -0,0 +1,8 @@
+import Controller from '@ember/controller';
+
+export default Controller.extend({
+  routing: Ember.inject.service('-routing'),
+  currentRouteName: Ember.computed.reads('routing.currentRouteName'),
+  checkRoute: Ember.computed.match('currentRouteName', /index/),
```

Comment:
contorllers do not need to have a routing computed property. please check on emberjs docs on how to access the route object from a controller. see if these methods can be accessed directly by the controller using controller methods.

Reply:
done. thanks for you help. Added ember truth-helper